config/Upgrade/services.yaml

```diff
@@ -98,9 +98,6 @@ services:
     send_report.step:
         class: Upgrade\Application\Strategy\Common\Step\SendReportStep
 
-    remove_integrator_lock_file.step:
```

Comment:
The idea to hold this step is to run integrator.lock cleaner once per upgrade. In this case, it will be executed per RG.  I'm not sure this is a good approach.

Reply:
this idea led to this bug

Reply:
I agree, but we already have some integrator prep steps in the upgrader, which confuses me a bit. 
Anyway, there's no objections from my side.